Packs/CrowdStrikeIntel/Integrations/CrowdStrikeFalconIntel_v2/CrowdStrikeFalconIntel_v2.py

```diff
@@ -66,7 +66,7 @@ def build_filter_query(self, args: dict[str, str]) -> str:
                 if key not in self.date_params:
                     values: List[str] = argToList(args[key], ",")
                     for value in values:
-                        filter_query += f"{key}:'{value}'+"
+                        filter_query += f'{key}:"{value}"+'
```

Comment:
The issue is that both `'` and `"` can appear in a URL so we might need a better handling